ksql-rest-app/src/main/java/io/confluent/ksql/rest/entity/ConnectorInfoEntity.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2019 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"; you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.rest.entity;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.Objects;
+import org.apache.kafka.connect.runtime.rest.entities.ConnectorInfo;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class ConnectorInfoEntity extends KsqlEntity {
```

Comment:
Can you mark `@Immutable` please, insurring `ConnectorInfo` is immutable.

Reply:
I cannot - `ConnectorInfo` is a connect class